modules/nf-core/bcftools/pluginimputeinfo/main.nf

```diff
@@ -0,0 +1,76 @@
+process BCFTOOLS_PLUGINIMPUTEINFO {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bcftools:1.20--h8b25389_1':
+        'biocontainers/bcftools:1.20--h8b25389_1' }"
+
+    input:
+    tuple val(meta), path(vcf), path(index)
+    path(regions)
+    path(targets)
```

Comment:
I can't myself evaluate this as I can't find out how to install bcftool plugins... can you confirm there are no other mandatory parameters, nor mandaotry or optional input files?